integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -83,6 +83,29 @@ def write_func(spark, path):
     assert_gpu_fallback_write(write_func, read_delta_path, data_path,
                               "ExecutedCommandExec", disable_conf)
 
+@allow_non_gpu("ExecutedCommandExec", *delta_meta_allow)
+@delta_lake
+@ignore_order
+@pytest.mark.skipif(is_before_spark_320(), reason="Delta Lake writes are not supported before Spark 3.2.x")
+@pytest.mark.xfail(is_databricks122_or_later(), reason="https://github.com/NVIDIA/spark-rapids/issues/8654")
+def test_delta_deletion_vector_fallback(spark_tmp_path):
+    data_path = spark_tmp_path + "/DELTA_DATA"
+    def setup_tables(spark):
+        setup_dest_tables(spark, data_path,
+                          dest_table_func=lambda spark: unary_op_df(spark, int_gen),
+                          use_cdf=True, enable_deletion_vectors=True)
```

Comment:
Saw that use_cdf was recently changed to True.  Curious, is that needed to get this test to pass?  Ideally we shouldn't care whether CDF is being used or not for this, and ideally be testing it both ways if there's a chance it could affect whether we are doing the right thing here.

Reply:
Ah, cdf stands for "change data feed", and not "create delta format". I thought it was used to trigger use of SQL to create the table. I will revisit this.